uwblueprint/childrens-aid-society
# Added dummy intake route (Julianne + Juthika)

## Notion ticket link

[Add dummy create /intakes route](https://www.notion.so/uwblueprintexecs/Add-dummy-create-intakes-route-26e861b0610b4a6f9f47f9d2e2f5473a)



## Implementation description
* created functions with dummy content for a dummy intake route



## Steps to test
1. run this command: `docker exec -it cas_py_backend /bin/bash -c "pip install -e . && pytest"` to make sure that all tests pass (which it should because it is hardcoded to pass)
2. run this command: `curl -X POST http://localhost:5000/intake` to make sure that the object returned is `{"status": "ok"}`




## What should reviewers focus on?
* nothing in particular, but maybe focus on the new files added


## Checklist
